pages/patterns/delete.js

```diff
@@ -1,108 +1,126 @@
+      4392908903
+    </Text>,
+    <Text as="span" fontFamily="fontFamilyText">
+      hi
+    </Text>,
+  ],
+  [
+    <Text as="span" fontFamily="fontFamilyText">
+      Second Thing
+    </Text>,
+    <Text as="span" fontFamily="fontFamilyCode">
+      8439025738
+    </Text>,
+    <Text as="span" fontFamily="fontFamilyText">
+      hello
+    </Text>,
+  ],
+  [
+    <Text as="span" fontFamily="fontFamilyText">
+      Third thing
+    </Text>,
+    <Text as="span" fontFamily="fontFamilyCode">
+      8439025738
+    </Text>,
+    <Text as="span" fontFamily="fontFamilyText">
+      Welcome!
+    </Text>,
+  ],
+  [
+    <Text as="span" fontFamily="fontFamilyText">
+      Fourth this
+    </Text>,
+    <Text as="span" fontFamily="fontFamilyCode">
+      6239485739
+    </Text>,
+  ],
+];
+
+export default function Delete() {
+  const [tableRows, setTableRows] = useState(rows);
 
   const toaster = useToaster();
   const [isOpen, setIsOpen] = React.useState(false);
-  const [serviceToDelete, setServiceToDelete] = React.useState({});
+  const [serviceToDelete, setServiceToDelete] = React.useState([]);
 
-  const deleteService = (serviceObj) => {
+  const deleteService = (serviceRow) => {
     toaster.push({
-      message: `${serviceObj.friendlyName.props.children} was successfully deleted.`,
+      message: `${serviceRow[0].props.children} was successfully deleted.`,
       variant: 'success',
       dismissAfter: 3000,
     });
-    const updatedServices = data.filter((service) => service.id !== serviceObj.id);
-    setData(updatedServices);
-    setServiceToDelete({});
+    const updatedServices = tableRows.filter((row) => tableRows.indexOf(serviceRow) !== tableRows.indexOf(row));
+    setTableRows(updatedServices);
+    setServiceToDelete([]);
   };
 
-  const handleDelete = (serviceObj) => {
+  const handleDelete = (serviceRow) => {
     if (severity === 'low') {
-      deleteService(serviceObj);
+      deleteService(serviceRow);
     } else if (severity === 'medium') {
       setIsOpen(true);
-      setServiceToDelete(serviceObj);
+      setServiceToDelete(serviceRow);
+    } else {
+      throw new Error(
+        'No severity level was detected. Go back to delete.js and make sure to change severity to "low", "medium", or "high"!',
+      );
     }
   };
 
   return (
     <>
-      <Toaster {...toaster} />
+      <Box marginBottom="space100">
+        <Alert variant="neutral">
+          <Text as="span">
+            <strong>Hello! </strong>To edit this page in CodeSandbox, go to the &quot;pages&quot; folder and click the
+            &quot;delete.js&quot; file. Check out the{' '}
+            <Anchor href="https://paste.twilio.design/patterns/delete/" showExternal>
+              Paste Delete Pattern documentation
+            </Anchor>{' '}
+            for more information. To get rid of this alert box, remove the {'"<Box></Box>"'} component from the bottom
+            of the &quot;delete.js&quot; file.
```

Comment:
I think this alert is going to be super helpful, and it's great that you give instructions on how to get rid of it for customizability!

You could make use of the <Code> component (imported differently from other Paste components- see it as one of the last imports in index.js) to differentiate the text when you refer to names in the file structure instead of quotes. I feel like it makes it more clear that we're talking about something in the code/file structure.

Also, these instructions carry over even outside CodeSandbox, so we probably just say "To edit this page, go to..."

Reply:
thanks for that suggestion. <Box> is now wrapped in <Code>. I agree that this will make it more apparent that the code needs to be edited to remove the alert. thanks for the help!!